elasticsearch-vmss/azuredeploy.json

```diff
@@ -0,0 +1,504 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterName": {
+      "type": "string",
+      "defaultValue": "es-azure",
+      "metadata": {
+        "description": "The name of the cluster."
+      }
+    },
+    "vmSku": {
```

Comment:
Which clouds is `Standard_D1_v2` not available in? The Azure pricing page shows this is in Germany and US Gov. 
Could you also explain why a variable is better than a parameter here?